torch/testing/_internal/inductor_utils.py

```diff
@@ -20,4 +20,4 @@
 ):
     pass
 
-HAS_CUDA = has_triton() and not TEST_WITH_ROCM
+HAS_CUDA = has_triton() and not TEST_WITH_ROCM
```

Comment:
I think this will make internal linter unhappy and you'll get reverted ;) cc @ZainRizvi what's the status on making that lint consistent?

Reply:
Linter has been acting weird today btw, these weird mypy errors showing up and it's also quite slow - not sure if thats' related

Reply:
Lint consistency work was merged in a couple weeks ago.

To take effect, it requires updating linter on the local dev machine by running `pip install lintrunner -U` (and you should be getting a warning/nudge to do so whenever you run `lintrunner` locally)